iroh/tests/batch.rs

```diff
@@ -0,0 +1,239 @@
+use std::{io, time::Duration};
+
+use bao_tree::blake3;
+use bytes::Bytes;
+use iroh::{
+    client::blobs::{AddDirOpts, WrapOption},
+    node::GcPolicy,
+};
+use iroh_blobs::store::mem::Store;
+
+async fn create_node() -> anyhow::Result<iroh::node::Node<Store>> {
+    iroh::node::Node::memory()
+        .gc_policy(GcPolicy::Interval(Duration::from_millis(10)))
+        .spawn()
+        .await
+}
+
+async fn wait_for_gc() {
+    // wait for multiple gc cycles to ensure that the data is actually gone
+    tokio::time::sleep(Duration::from_millis(50)).await;
+}
```

Comment:
Is there no other way of getting notified when GC has run or perhaps even triggering GC ourselves?
I really don't want yet another suite of tests that runs 50+ seconds :(

Reply:
It's 50 milliseconds. Sure, I can hook up some callbacks, but I think 50ms is fine...

Reply:
We have [`iroh::node::Builder::register_gc_done_cb`](https://n0-computer.github.io/iroh/pr/2545/docs/iroh/node/struct.Builder.html#method.register_gc_done_cb) can we use this here instead?
I think this may be more robust against slow machines, bad CI days and such.